src/components/Compare/CompareTable.tsx

```diff
@@ -108,23 +107,26 @@ const CompareTable = (props: {
     return [...sortedLocations, ...locationsWithoutValue];
   }
 
+  const columns = vaccinesFirst ? orderedColumnsVaccineFirst : orderedColumns;
+
   const getPopulation = (location: SummaryForCompare) =>
     location.region.population;
-  const getMetricValue = (location: any) => {
+  const getSortByValue = (location: SummaryForCompare) => {
     // TODO(https://trello.com/c/x0G7LZ91): Not sure if this check should be necessary,
     // but we seem to be missing projections for Northern Islands Municipality, MP right now.
     if (!location.metricsInfo) {
       return null;
     }
-    return location.metricsInfo.metrics[sorter].value;
+    let sortedColumn = columns.find(c => c.columnId === sorter) ?? columns[0];
```

Comment:
`sorter` used to always be a metric number, but now we have non-metric columns, so I assigned each column an ID. 🤷